react/features/chat/components/native/ChatTabBar.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+
+import { createMaterialTopTabNavigator } from '@react-navigation/material-top-tabs';
+import React, { PureComponent } from 'react';
+import { Dimensions } from 'react-native';
+
+import { Chat } from '../../../chat';
+import { chatTabBarOptions } from '../../../conference/components/native/constants';
+import { PollsPane } from '../../../polls/components';
+
+/**
+ * Class to implement the navigation for the chat screen, between tabs.
+ */
+class ChatTabBar extends PureComponent<*> {
+    /**
+     * Implements {@code Component#render}.
+     *
+     * @inheritdoc
+     */
+    render() {
+        const ChatTab = createMaterialTopTabNavigator();
+
+        return (
+            <ChatTab.Navigator
+                initialLayout = {{
```

Comment:
Why is this necessary? In general we try to avoid things like this because on tablets it's possible to resize apps when in split view for example.

Reply:
Passing this will improve the initial rendering performance. For most apps, this is a good default.

Reply:
Ack, since it's just the initial layout it's probably fine.